libs/genai/pyproject.toml

```diff
@@ -46,6 +54,10 @@ codespell = "^2.2.0"
 optional = true
 
 [tool.poetry.group.test_integration.dependencies]
+numexpr = { version = "^2.8.8", python = ">=3.9,<4.0" }
+google-api-python-client = "^2.114.0"
+google-cloud-datastore = "^2.19.0"
```

Comment:
we don't use this library in this project

Reply:
@lkuligin 
I updated PR. Remove unused libs.